presto-main/src/main/java/com/facebook/presto/sql/analyzer/StatementAnalyzer.java

```diff
@@ -2254,7 +2264,10 @@ private boolean analyzeLimit(FetchFirst node)
                 analysis.setLimit(node, rowCount);
             }
 
-            return node.isWithTies();
+            if (node.isWithTies()) {
+                return true;
+            }
+            return false;
```

Comment:
Is this necessary?

Reply:
this is part of implementation for Fetch first with Ties that needs to analylze if sementically correct when limit node has Ties presented, needed

Reply:
I meant, could we simply `return node.isWithTies()`?

Reply:
righty, fixed